packages/client/components/GlobalBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx'
+import React from 'react'
+
+interface Props {
+  bgColor: string
+  color: string
+  text: string
+}
+
+// https://www.astrouxds.com/components/classification-markings/#banner-examples
+
+// Note: banner height is 24px per Tailwind settings (Scale of 6: h-6, leading-6, p-6, etc.)
+// This height value is also in constEnums GlobalBanner
+
+const GlobalBanner = (props: Props) => {
+  const {bgColor, color, text} = props
+  return (
+    <div className='w-100 h-6 max-h-6 pt-6'>
```

Comment:
@mattkrick I added a container here (reducing a container from Action.tsx). This pushes the container down that is in Action and use height instead of padding which makes the inner containers that are based on height behave nicely.